rules/integrations/aws/persistence_redshift_instance_creation.toml

```diff
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2022/04/12"
+maturity = "production"
+updated_date = "2022/04/12"
+integration = "aws"
+
+[rule]
+author = ["Elastic", "Pete Hampton"]
+description = "Identifies the creation of an Amazon Redshift cluster."
```

Comment:
We could elaborate a bit on why it is important to monitor this, usages on the attacker's perspective, etc.

Reply:
The rule I really want to write is around the networking around redshift (a dw that we are likely to use soon) - https://docs.aws.amazon.com/redshift/latest/APIReference/API_AuthorizeClusterSecurityGroupIngress.html
However, surveying your other rules, particularly around RDS clusters / instances, there is rules around the creation and deletion of the data stores.

Should I close this and just write the security group ingress changes?